tools/operator/testclient/operations/invokequery.go

```diff
@@ -261,20 +294,25 @@ func (i InvokeQueryUIObject) invokeQueryTransactions(invokeQueryObjects []Invoke
 					blockHash = value.peerBlockHash
 				} else {
 					if value.peerBlockchainHeight != blockchainHeight || value.peerTransactionCount != transactionCount || value.peerBlockHash != blockHash {
-						logger.ERROR("Peers are not in sync")
-						statusError = fmt.Errorf("Something went wrong with peer %s as blockchain height or transactions does not match up", key)
+						logger.ERROR("Peers are not in sync %s", key)
+						r <- Result{err}
 					}
 				}
 			}
-		}(key, &wg)
+		}(key, &wg, c)
 	}
 	wg.Wait()
-	return statusError
+	close(c)
+	r := <-c
+	if r.Err != nil {
+		return r.Err
+	}
+	return nil
```

Comment:
Instead of just returning just the one error, can we pull all of the errors out of the channel, log them, and then return a generic, `failed fetching metrics` error.

Otherwise this looks good.